.github/workflows/golangci-lint.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: "golangci-lint"
+on:
+  - push
+  - pull_request
+
+jobs:
+  golangci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
+        with:
+          go-version: '1.19'
+          cache: false
+      - name: lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          version: v1.54
```

Comment:
The latest is `v1.55.0` according to [official page](https://github.com/golangci/golangci-lint/releases). They update pretty often. Can we try to go with `v1.55`?